remote.go

```diff
@@ -44,6 +44,16 @@ var remoteErrors = map[int]string{
 	32: "invalid selector",
 }
 
+// Alias for a type that is passed as an argument for selenium.Wait
```

Comment:
Also document what the behavior of the Wait.* functions are with respect to the return values.